apps/prairielearn/src/lib/externalGraderLocal.ts

```diff
@@ -205,7 +216,7 @@ export class ExternalGraderLocal {
  *
  * On Windows, we use $USERPROFILE instead of $HOME.
  */
-function getDevJobDirectory(jobId) {
+function getDevJobDirectory(jobId: string): string {
```

Comment:
Good observation! This is totally broken. I'm inclined to say that `JOBS_DIR` is unused and can be removed. Otherwise, if we want to keep it, we should fix it to append `/job_${jobId}` at the end of the path.

Reply:
Fixed in 7c54cf4cf3d62fbc103c8d5473b0dedd7de0104b.